plugins/importexport/onix30/filter/MonographONIX30XmlFilter.php

```diff
@@ -586,6 +588,32 @@ public function createProductNode(DOMDocument $doc, Submission $submission, Publ
 
         $websiteNode->appendChild($this->buildTextNode($doc, 'WebsiteLink', $request->url($context->getPath(), 'catalog', 'book', [$submissionBestId])));
 
+        /* --- Funders and awards --- */
+        $fundingData = $plugin->getFundingData($context->getId(), $submissionBestId);
```

Comment:
$submissionBestId can also be the urlPath, but the function needs the numeric ID